mvp-spec/audit-log-design.md

```diff
@@ -135,14 +98,56 @@ the Contracting Party handles those objects.
+```
+</details>
 
-Once the user has clicked on the Audit Button, she/he is redirected to the
-Audit UI. The Contracting Party generates the UI based on the validations 
-of the Prebid SSO Data and the Transmission Results. The following elements must
+###  Alternative 2 - Javascript
+
+The Audit logs can be stored on the `window` of the webpage via `javascript`. The exact path is `window.paf.audit`. The `audit` object is a dictionary that provides the audit log (the value) given the Transaction Id (the key). 
+
+```javascript
+window.paf = {};
```

Comment:
NIT: since you are creating a paf object that might store other data than the audit, be careful not to remove potential other data, and favor "append", such as:

`window.paf = window.paf ?? {}`